FreeSurfer/scripts/FreeSurferHiresPial.sh

```diff
@@ -75,13 +76,20 @@ cp --preserve=timestamps $SubjectDIR/$SubjectID/surf/rh.pial $SubjectDIR/$Subjec
 cp --preserve=timestamps $SubjectDIR/$SubjectID/surf/lh.thickness $SubjectDIR/$SubjectID/surf/lh.thickness.preT2.pass1
 cp --preserve=timestamps $SubjectDIR/$SubjectID/surf/rh.thickness $SubjectDIR/$SubjectID/surf/rh.thickness.preT2.pass1
 
-# Generate pial surfaces with T2 adjustment, still first pass.  
-# Same 4 files created as above, but have $outputSuffix1 ("postT2.pass1") appended through use of -output flag.
-# Use -T2 flag (rather than -T2dura), since -T2 is the flag used within recon-all script of FS 5.3 (but -T2 and -T2dura generate same results)
-#For mris_make_surface with correct arguments #Could go from 3 to 2 potentially...
-outputSuffix1=".postT2.pass1"
-mris_make_surfaces -nsigma_above 2 -nsigma_below 3 -aseg aseg.hires -filled filled.hires -wm wm.hires -mgz -sdir $SubjectDIR -orig white.deformed -nowhite -orig_white white.deformed -orig_pial pial -T2 "$mridir"/T2w_hires.norm -T1 T1w_hires.norm -output $outputSuffix1 $SubjectID lh
-mris_make_surfaces -nsigma_above 2 -nsigma_below 3 -aseg aseg.hires -filled filled.hires -wm wm.hires -mgz -sdir $SubjectDIR -orig white.deformed -nowhite -orig_white white.deformed -orig_pial pial -T2 "$mridir"/T2w_hires.norm -T1 T1w_hires.norm -output $outputSuffix1 $SubjectID rh
+if [ ! "${T2wImage}" = "NONE" ] ; then
+  verbose_red_echo "---> Generating pial first pass surfaces with T2 adjustment."
+  # Generate pial surfaces with T2 adjustment, still first pass.
+  # Same 4 files created as above, but have $outputSuffix1 ("postT2.pass1") appended through use of -output flag.
+  # Use -T2 flag (rather than -T2dura), since -T2 is the flag used within recon-all script of FS 5.3 (but -T2 and -T2dura generate same results)
+  # For mris_make_surface with correct arguments #Could go from 3 to 2 potentially...
+  outputSuffix1=".postT2.pass1"
+  mris_make_surfaces -nsigma_above 2 -nsigma_below 3 -aseg aseg.hires -filled filled.hires -wm wm.hires -mgz -sdir $SubjectDIR -orig white.deformed -nowhite -orig_white white.deformed -orig_pial pial -T2 "$mridir"/T2w_hires.norm -T1 T1w_hires.norm -output $outputSuffix1 $SubjectID lh
+  mris_make_surfaces -nsigma_above 2 -nsigma_below 3 -aseg aseg.hires -filled filled.hires -wm wm.hires -mgz -sdir $SubjectDIR -orig white.deformed -nowhite -orig_white white.deformed -orig_pial pial -T2 "$mridir"/T2w_hires.norm -T1 T1w_hires.norm -output $outputSuffix1 $SubjectID rh
+else
+  verbose_red_echo "---> No T2w image, skipping generation of pial first pass surfaces with T2 adjustment."
+  outputSuffix1=""
+fi
+
 
 # Bring pial surfaces out of highres space into the 1 mm (FS conformed) space
 # [Note that $regII, although named 1mm2hires.dat, actually maps from the hires space into the FS conformed space].
```

Comment:
Are you sure that setting `outputSuffix1=""` in the case of no T2 in the old FS5.3 version code isn't breaking something given that the immediately following `mri_surf2surf` command operates on `pial${outputSuffix1}`, which will then resolve to just `pial`, which is itself is a pretty elemental FS file... ?

Reply:
@mharms, based on my read, the first pass w/o T2w adjustments generates `lh/rh.pial` surfaces. These are then copied for debugging purposes to `.preT2.pass1`. What else could `mri_surf2surf` work with?

Reply:
So, in the no T2 case, `--sval-xyz` and `--tval` are both set to 'pial', although I guess that is fine as long they are distinct copies within memory of `mri_surf2surf` (which I would assume to be the case...)  (i.e., that it reads in `--sval-xyz` entirely before writing out `--tval`).

Reply:
I can change that to the stored copy.

Reply:
Just to be sure.

Reply:
Changed.